src/xenia/gpu/gl4/command_processor.cc

```diff
@@ -2849,8 +2849,8 @@ bool CommandProcessor::IssueCopy() {
   // TODO(benvanik): figure out real condition here (maybe when color cleared?)
   // HACK: things seem to need their depth buffer cleared a lot more
   // than as indicated by the depth_clear_enabled flag.
-  // if (depth_clear_enabled) {
-  if (depth_target != kAnyTarget) {
+  if (depth_clear_enabled) {
+    // if (depth_target != kAnyTarget) {
```

Comment:
Yep, i will then do it as 

          if (depth_clear_enabled && depth_target != kAnyTarget) {